lago/sdk.py

```diff
@@ -0,0 +1,82 @@
+from __future__ import print_function
+from lago import cmd
+from sdk_utils import SDKWrapper
+
+
+def init(config, workdir=None, **kwargs):
+    """
+    Initialize the Lago environment
+
+    Args:
+        config(str): Path to LagoInitFile
+        workdir(str): Path to initalize the workdir
+        **kwargs(dict): Pass arguments to :func:`~lago.cmd.do_init`
+
+    Returns:
+        :class:`~lago.sdk.sdk`: Initialized Lago enviornment
+
+    Raises:
+       :exc:`~lago.utils.LagoException`: If initialization failed
+    """
+
+    # .. to-do:: load global configuration from a file
+    # .. to-do:: allow loading the env from an existing directory
+    defaults = {
+        'workdir': workdir,
+        'template_repo_path': 'http://templates.ovirt.org/repo/repo.metadata',
+        'template_store': '/var/lib/lago/store',
+        'template_repos': '/var/lib/lago/repos',
+        'set_current': True,
+        'skip_bootstrap': False,
+        'skip_build': False,
+        'prefix_name': 'default',
+    }
+    defaults.update(kwargs)
+    defaults['virt_config'] = config
+    workdir, prefix = cmd.do_init(**defaults)
+    return sdk(workdir, prefix)
+
+
+class sdk(object):
+    """
+    The SDK can be initialized in 2 ways:
+    1. (Preferred) - by calling :func:`sdk.init`.
+    2. By passing already created workdir and prefix objects.
```

Comment:
+1 - on my to do list - let initialize an environment from an existing path. Will formulate all these to-dos into an issue.